Packs/OPSWAT-Filescan/pack_metadata.json

```
@@ -0,0 +1,20 @@
+{
+    "name": "OPSWAT Filescan",
+    "description": "Unique adaptive threat analysis technology",
+    "support": "xsoar",
```

```suggestion
    "support": "partner",
```
I suppose this is a mistake and this contribution is supported by you - if this is not the case, please let us know.

---

Thanks! You are right. I corrected the pack_metadata file